### ISSUE TITLE
Accessibility Audit: Unable to differentiate button and links in Grayscale

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#948]
On a Mac, turn on Gray scale mode (System Preferences -> Accessibility -> Display -> Color Filters -> 'Enable Color Filters'). Navigate to a [Listing Detail](https://sites-public-tj3gg4i5eq-uc.a.run.app/listing/ff945428-fcd7-46bf-a290-3c9219039df1/oak_village_independence_4401_burlingame_st_detroit_mi) page. In a regular display, buttons/links has blue font color to differentiate that section from other elements in the screen. But in grayscale mode, the user is unable to differentiate and it does not appear to be a link or button. There is no separate indication (such as a button border) to differentiate buttons/links from other text on screen.
